eng/Versions.props

```diff
@@ -272,6 +273,7 @@
     <PrivateVisualStudioPackage Include="VSLangProj80" />
     <PrivateVisualStudioPackage Include="VSLangProj140" />
     <PrivateVisualStudioPackage Include="VsWebsite.Interop" />
+    <PrivateVisualStudioPackage Include="Microsoft.VSSDK.Debugger.VSDConfigTool" />
```

Comment:
><PrivateVisualStudioPackage Include="Microsoft.VSSDK.Debugger.VSDConfigTool" /> [](start = 4, length = 79)

This shouldn't be necessary since you already set `PrivateAssets=all` on the PackageReference.